tests/Executor/DeferredFieldsTest.php

```diff
@@ -401,15 +401,15 @@ public function testComplexRecursiveDeferredFields() : void
                 return [
                     'sync'         => [
                         'type'    => Type::string(),
-                        'resolve' => function ($v, $a, $c, ResolveInfo $info) {
+                        'resolve' => function ($val, $args, $context, ResolveInfo $info) {
```

Comment:
I think we can use `value` and `arguments` if we are doing this already :)

Reply:
`args` is used anywhere, and `$val` is going to be `$rootValue`